lib/install/config/webpacker.yml

```diff
@@ -0,0 +1,37 @@
+# Note: You must restart bin/webpack-dev-server for changes to take effect
+
+default: &default
+  dev_server_host: 0.0.0.0
+  dev_server_port: 8080
+  dev_server_https: false
```

Comment:
Thinking more.. Maybe we should move these options to the `development` config only since `bin/webpack-dev-server` exclusive loads `config/webpack/development.js` (which is the only config file containing `devServer`).
```yaml
default: &default
  source_path: app/javascript
  source_entry_path: packs
  public_output_path: packs
  ...
  
development:
  <<: *default

  dev_server:
    host: 0.0.0.0
    port: 8080
    https: false
...
```